### PR TITLE
Amazon2 tweaks

### PR DESCRIPTION
**Pull request by Shplorf**
- Address issue where Amazon Linux 2 is not always correctly detected
- Address issue where auditd is not stopped on Amazon Linux 2

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@
   include: pkg_url.yml
   when: threatstack_pkg_url is undefined
 
+- name: Disable auditd service
+  include: disable_auditd.yml
+  when: ansible_distribution == 'Amazon'
```
Comment (pedroMMM) [inline tasks/main.yml]: @Shplorf this should only be applied to Amazon Linux 2.

File: tasks/disable_auditd.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  ignore_errors: true
+  changed_when: false
+
+- name: Stop service auditd
+  command: service auditd stop
+  when: auditd_status.rc == 0
+
+- name: Disable service auditd
+  command: systemctl disable auditd
+  when: auditd_status.rc == 0
```
Comment (pedroMMM) [inline tasks/disable_auditd.yml]: Shouldn't it be `when: auditd_status.rc != 0`?

File: tasks/main.yml (hunk 1 above)
Comment (pedroMMM) [reply tasks/main.yml]: I realized my mistake and the correct fix is on: https://github.com/threatstack/threatstack-ansible/pull/48/files#r255199810

**Comment by olhado** [comment]
----- Original message -----

**Pull request merged**